nilearn/input_data/tests/test_base_masker.py

```diff
@@ -3,12 +3,12 @@
 """
 
 import numpy as np
-from numpy.testing import assert_array_almost_equal
+from numpy.testing import assert_array_almost_equal, assert_raises
```

Comment:
Unless I am missing something, assert_raises is not used in test_base_masker.py.